stdlib/ballerina-builtin/src/main/ballerina/internal/exec_ballerina.bal

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2017-2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change this to 2018?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/9227/commits/9b2f2ce167828ea857fabb067b03d97ac5c4f139